### PR TITLE
Browsi Bid Adapter: initial release

### PR DESCRIPTION
**Pull request by itamaravady-browsi**
## Type of change

- [ x] New bidder adapter  
-  

## Description of change
Submitting a new bid adapter named "Browsi Bid Adapter". This adapter only supports Video file type.

support@browsi.com
itamarav@browsi.com

- test parameters for validating bids:
```
let videoAdUnit = [
    {
        code: 'videoAdUnit',
        mediaTypes: {
            video: {
                playerSize: [[300, 250]],
                context: 'outstream'
            },
        },
        bids: [{
            bidder: 'browsi',
            params: {
                pubId: '117a476f-9791-4a82-80db-4c01c1683db0', // Demo Publisher ID
                tagId: '1' // Demo Tag ID
            }
        }]
    }];
```

## Other information
related pr: https://github.com/prebid/prebid.github.io/pull/4377

### CODE REVIEW

File: modules/browsiBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        url: params.endpoint || ENDPOINT,
+        data: {
+          requestId: bidderRequestId,
+          bidId: bidId,
+          timeout: getTimeout(bidderRequest),
+          baData: getData(),
+          referer: refererInfo.page || refererInfo,
+          gdpr: gdprConsent,
+          ccpa: uspConsent,
+          sizes: sizes,
```
Comment (lksharma) [inline modules/browsiBidAdapter.js:55]: In the case of hybrid AdUnits (banner + video), sizes will refer to the banner size. If you would like to have `sizes` reference the video size you should use `video.playerSize` here.

File: modules/browsiBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    .replace(GDPR_STR_TO_REPLACE, encodeURIComponent(gdprConsent?.consentString || ''))
+    .replace(USP_TO_REPLACE, encodeURIComponent(uspConsent?.consentString || ''));
+  if (validUrl.indexOf('http') < 0) {
+    validUrl = 'http://' + validUrl;
+  }
+  return validUrl;
+}
+
+const validateBrowsiIds = function (pubId, tagId) {
+  return isStr(pubId) && isStr(tagId);
```
Comment (lksharma) [inline modules/browsiBidAdapter.js:160]: `pubId` & `tagId` are required but they can also be empty strings. Is the expected?

File: modules/browsiBidAdapter.js (hunk 1 above)
Comment (itamaravady-browsi) [reply modules/browsiBidAdapter.js]: thank you - fixed

File: modules/browsiBidAdapter.js (hunk 2 above)
Comment (itamaravady-browsi) [reply modules/browsiBidAdapter.js]: its not expected - fixed

**Review by lksharma — APPROVED** [review]
LGTM

**Pull request merged**